rest-api/model/organization.py

```diff
@@ -15,3 +15,4 @@ class Organization(Base):
   hpoId = Column('hpo_id', Integer, ForeignKey('hpo.hpo_id'), nullable=False)
   # Sites belonging to this organization.
   sites = relationship('Site', cascade='all, delete-orphan', order_by='Site.googleGroup')
+  isObsolete = Column('is_obsolete', SmallInteger, default=0)
```

Comment:
You should be able to use Enum(ObsoleteStatus) here. See:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/model/participant.py#L43

You'll need to change script.py.mako or the generated migrations to import the enum class, e.g.

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/alembic/script.py.mako#L18